src/query/execute_schema.c

```diff
@@ -5357,6 +5360,9 @@ do_create_partition_constraint (PT_NODE * alter, SM_CLASS * root_class, SM_CLASS
       for (; parts; parts = parts->next)
 	{
 	  MOP subclass_op = parts->info.parts.name->info.name.db_object;
+	  SM_INDEX_STATUS old_status = SM_NORMAL_INDEX;
```

Comment:
In fact, `old_status` is unused and I don't think it is needed. I think to pass `constraint->index_status` would be enough.